app/controllers/completed_transaction_controller.rb

```diff
@@ -0,0 +1,37 @@
+require "slimmer/headers"
+
+class CompletedTransactionController < ApplicationController
+  before_filter :redirect_if_api_request
+  before_filter -> { set_expiry unless viewing_draft_content? }
+
+  LEGACY_SLUGS = ["transaction-finished", "driving-transaction-finished"].freeze
```

Comment:
it's not at all obvious why this is here. Could there be a comment to explain? It will be tricky to investigate the git history of this line, once this is merged.